airbyte-integrations/connectors/source-mongodb-v3/acceptance-test-config.yml

```diff
@@ -0,0 +1,29 @@
+# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-mongodb-v2:dev
```

Comment:
shouldn't this be `v3`?

Reply:
@colesnodgrass Yes...good catch. Now its `-internal-poc`.